test/cppsuite/tests/run.cxx

```diff
@@ -121,8 +124,11 @@ run_test(const std::string &test_name, const std::string &config, const std::str
         hs_cleanup(test_harness::test_args{config, test_name, wt_open_config}).run();
     else if (test_name == "burst_inserts")
         burst_inserts(test_harness::test_args{config, test_name, wt_open_config}).run();
-    else if (test_name == "search_near_01")
-        search_near_01(test_harness::test_args{config, test_name, wt_open_config}).run();
+    /*
+     * Disabled as part of WT-8909.
+     * else if (test_name == "search_near_01")
+     *   search_near_01(test_harness::test_args{config, test_name, wt_open_config}).run();
```

Comment:
Should we just remove these instead of commenting? Let me know what your thoughts are @etienneptl

Reply:
I removed them instead of commenting out.